apps/desktop/src/renderer/routes/_authenticated/_dashboard/v2-workspace/$workspaceId/hooks/usePaneRegistry/components/ChatPane/components/WorkspaceChatInterface/components/ChatMessageList/components/SubagentExecutionMessage/SubagentExecutionMessage.tsx

```
@@ -62,46 +62,32 @@ export function SubagentExecutionMessage({
-								labelClassName="flex cursor-pointer items-center gap-2 text-xs text-muted-foreground"
-								markdownContainerClassName="max-h-[32rem] overflow-auto rounded border bg-background/80 p-2"
-								plainContainerClassName="max-h-[32rem] overflow-auto rounded border bg-background/80 p-2 text-xs whitespace-pre-wrap break-words"
-							/>
-						) : null}
 						{subagent.toolCalls.length > 0 ? (
-							<div className="flex flex-wrap items-center gap-1.5">
+							<div className="space-y-1">
 								{subagent.toolCalls.map((tool, index) => (
-									<span
+									<SubagentInnerToolCall
 										key={`${subagent.toolCallId}-${tool.name}-${index}`}
```

Biome (2.4.10)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)